### ISSUE TITLE
Ability to quote from multiple messages, or to quote only part of a message

### ISSUE THREAD

**Issue by DMRobertson** [vector-im/element-meta#365]
#### What would you like to do?

1. Click and drag over multiple messages
2. Pop up menu -> "..." -> "Quote"
3. Populate my text entry field with quotes from the text selected

#### Why would you like to do it?

I only want to respond to specific parts of text in a sequence of many short messages.

Or: I only want to respond to specific bits of text in one long message.

### Additional context

Basically I want what GitHub has. Click and drag a bunch of text on an issue -> "..." menu -> "quote reply".

**Comment by DMRobertson** [comment]
The behaviour at present: you can select whatever you like and try to quote it, but it will only quote (the entirety of) the message you happen to have your mouse cursor hovering over.

**Comment by HarHarLinks** [comment]
> Populate my text entry field with quotes from the text selected

would be nice to have this formatted properly, linking back to the respective author (and message?)

In particular this could be made to integrate well with the currently labs "Right-click message context menu".
